### ISSUE TITLE
`!.[true]` errors rather than broadcasting `!`

### ISSUE THREAD

**Issue by Sacha0** [JuliaLang/julia#20039]
```julia
julia> versioninfo()
Julia Version 0.6.0-dev.2075
Commit fe1f663* (2017-01-14 01:37 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin15.6.0)
  CPU: Intel(R) Core(TM) i7-3520M CPU @ 2.90GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, ivybridge)

julia> !.[true]
ERROR: syntax: invalid identifier name "."
```
Expected behavior: `broadcast(!, [true])`. Ref. https://github.com/JuliaLang/julia/issues/20037#issuecomment-272647370. Best!

**Comment by nalimilan** [comment]
Should be `.!` instead, right? Doesn't work either:
```julia
julia> .!
ERROR: syntax: invalid operator ".!"
```

**Comment by Sacha0** [comment]
> Should be .! instead, right?

I've seen `!` only as prefix unary, whereas dot-ops are typically infix binary? Best!

**Comment by stevengj** [comment]
If I recall correctly, unary dot operators are not yet implemented.  (`√.[1,2,3]` gives a similar error.)

**Comment by stevengj** [comment]
`(!).([true])` works, of course.

**Comment by Sacha0** [comment]
Closed by #20249 (with `.!` rather than `!.`). Best!
